docs/getting_started/ros_interface/ros2.rst

```diff
@@ -2,69 +2,91 @@
 
-Install ROS 2
-=============
+Installing ROS 2
+================
 
 .. note::
 
-  The ROS 2 Interbotix SLATE packages are only compatible with Humble on Ubuntu 22.04.
+  The ROS 2 Interbotix SLATE packages are compatible with the following distributions: Galactic, Humble, and Iron on Ubuntu 22.04, and Jazzy and Rolling on Ubuntu 24.04.
 
-Follow the `Ubuntu (Debian packages)`_ ROS 2 installation guide on the ROS documentation site.
-Once installed, make sure to follow the `How do I use the rosdep tool?`_ guide to to set up and install the development and dependency toolchains.
+Follow the ROS 2 installation guide for your specific version on the ROS documentation site:
 
-.. _`Ubuntu (Debian packages)`: https://docs.ros.org/en/humble/Installation/Ubuntu-Install-Debians.html
-.. _`How do I use the rosdep tool?`: https://docs.ros.org/en/humble/Tutorials/Intermediate/Rosdep.html#how-do-i-use-the-rosdep-tool
+- `Galactic`_ (Ubuntu 22.04)
+- `Humble`_ (Ubuntu 22.04)
+- `Iron`_ (Ubuntu 22.04)
+- `Jazzy`_ (Ubuntu 24.04)
+- `Rolling`_ (Ubuntu 24.04)
 
-Install the Interbotix ROS Slate Packages
-=========================================
+.. _Galactic: https://docs.ros.org/en/galactic/Installation/Ubuntu-Install-Debians.html
+.. _Humble: https://docs.ros.org/en/humble/Installation/Ubuntu-Install-Debians.html
+.. _Iron: https://docs.ros.org/en/iron/Installation/Ubuntu-Install-Debians.html
+.. _Jazzy: https://docs.ros.org/en/jazzy/Installation/Ubuntu-Install-Debians.html
+.. _Rolling: https://docs.ros.org/en/rolling/Installation/Ubuntu-Install-Debians.html
 
-#.  Clone the humble branch of the interbotix_ros_core repository into your colcon workspace.
+Configuring Port Permissions
+============================
 
-  .. code-block:: bash
+The SLATE uses a USB-Serial converter to communicate with the host computer.
 
-    $ mkdir -p ~/interbotix_ws/src
-    $ cd ~/interbotix_ws/src
-    $ git clone -b humble https://github.com/Interbotix/interbotix_ros_core.git
+Add your user to the dialout group:
 
-#.  Remove the COLCON_IGNORE file preventing the interbotix_ros_slate metapackage from being built.
+.. code-block:: bash
+
+  $ sudo usermod -a -G dialout $USER
+
+Reboot your computer to apply the changes:
 
-  .. code-block:: bash
+.. code-block:: bash
 
-    $ rm ~/interbotix_ws/src/interbotix_ros_core/interbotix_ros_slate/COLCON_IGNORE
+  $ sudo reboot
 
-#.  Run rosdep in the root of the workspace to install all required dependencies.
+Installing the Interbotix ROS SLATE Packages
+============================================
 
-  .. code-block:: bash
+Clone the repository and remove the COLCON_IGNORE file:
 
-    $ cd ~/interbotix_ws
-    $ rosdep install --from-paths src --ignore-src -r -y
+.. code-block:: bash
 
-#.  Build the workspace, making sure that you have sourced your ROS system installation first.
+  $ mkdir -p ~/interbotix_ws/src
+  $ cd ~/interbotix_ws/src
+  $ export ROS_DISTRO=humble  # Replace 'humble' with your desired ROS distribution
```

Comment:
Assuming they already have ROS installed and have pre-sourced their ROS install, the `ROS_DISTRO` variable would be set